docs/dev/coding_standards.rst

```diff
@@ -19,9 +19,14 @@ Many editors/IDEs can run this tool in the background while you work, showing re
 
 .. _flake8: https://flake8.pycqa.org/en/latest/
 
-It is allowed but not required to use the `black`_ code formatter. Do only format files that you are actively working on and make a dedicated commit for the initial formatting when working on an existing file that does not adhere to the black code style to facilitate code review. If there are conflicts between black's output and flake8 (especially related to `E203`_), flake8 takes precedence. Use ``#noqa : E203`` to disable E203 warnings for a specific line if appropriate.
+It is allowed but not required to use the `black`_ code formatter. 
+To avoid introducing unrelated changes when working on an existing file, it is recommended to use the `darker`_ tool.
```

Comment:
```suggestion
To avoid introducing unrelated changes when working on an existing file, it is recommended to use the `darker`_ tool instead of `black`.
```
We do not recommend darker in itself, but recommend it as a replacement for black, don't we?